src/utils/components/DeveloperPreviewLabel/DeveloperPreviewLabel.tsx

```diff
@@ -15,7 +13,7 @@ const DeveloperPreviewLabel: React.FC = () => {
         'Developer preview features are not intended to use in production environments. The clusters deployed with the developer preview features are developmental clusters and are not currently supported by Red Hat.',
       )}
     >
-      <Label isCompact icon={<InfoCircleIcon />} className="dev-preview-label">
+      <Label isCompact icon={<InfoCircleIcon />} color="orange">
```

Comment:
I remember someone was bad to me some time ago exactly because of this change and wanted me to put it all to a separate css file, haha 🥲

Reply:
The preferred way is external css, as we don't do inline styling as it is tough to override when needed, so the color prop is the same. It is more like a pattern fly issue that, on theme change, they don't change the var color. Another solution for this bug can be adding a selector for a dark theme and changing to a different color aviv

Reply:
@hstastna please don't approve PR's while I requested more info just 10 minutes before you approved. it's not nice.